examples/getting_started/README.md

```diff
@@ -162,16 +188,31 @@ The command can be broken down into the following parts:
 
 Already during the training, the checkpoints can be found locally in `checkpoints/` and the loss and metric developments can be inspected online in [Weights&Biases](https://wandb.ai/). 
 
-## Evaluation
+### Evaluation
 
-Given a checkpoint and tokenizer, we can load the model for text generation as follows
+In order to let the model generate text, we need to specify the last training checkpoint under `model_path` in the config file `example_text_generation_config.yaml`:
+
+```
+# example_text_generation_config.yaml
+
+settings:
+  referencing_keys:
+    sample_key: input_ids
+    prediction_key: logits
+  model_path: ./checkpoints/<checkpoint_name>.bin
+  device: 0
+  context_length: 1024
```

Comment:
maybe 512?

Reply:
Fixed in e391cfc2646dcb52e9943449cc42f665f3b3b5fc.